doc/quickstart.rst

```diff
@@ -58,7 +58,7 @@ In cases, when no agent was forwarded, SSH-MITM can rediredt the session to a ho
                         <p>
                             To start an intercepting mitm-ssh server on Port 10022,
                             all you have to do is run a single command.<br/>
-                            <code>$ ssh-mitm --remote-host 192.168.0.x</code>
+                            <code>$ ssh-mitm --remote-host 192.168.0.x:PORT</code>
```

Comment:
we should remove the port part in the quickstart section and mention, that port 22 will be used for the remote server